doc/api/querystring.md

```diff
@@ -13,6 +13,56 @@ query strings. It can be accessed using:
 const querystring = require('querystring');
 ```
 
+## querystring.decode(str[, sep[, eq[, options]]])
+<!-- YAML
+added: v0.1.25
+changes:
+  - version: v8.0.0
+    pr-url: https://github.com/nodejs/node/pull/10967
+    description: Multiple empty entries are now parsed correctly (e.g. `&=&=`).
+  - version: v6.0.0
+    pr-url: https://github.com/nodejs/node/pull/6055
+    description: The returned object no longer inherits from `Object.prototype`.
+  - version: v6.0.0, v4.2.4
+    pr-url: https://github.com/nodejs/node/pull/3807
+    description: The `eq` parameter may now have a length of more than `1`.
+-->
+
+* `str` {string} The URL query string to parse
+* `sep` {string} The substring used to delimit key and value pairs in the
+  query string. **Default:** `'&'`.
+* `eq` {string}. The substring used to delimit keys and values in the
+  query string. **Default:** `'='`.
+* `options` {Object}
+  * `decodeURIComponent` {Function} The function to use when decoding
+    percent-encoded characters in the query string. **Default:**
+    `querystring.unescape()`.
+  * `maxKeys` {number} Specifies the maximum number of keys to parse.
+    Specify `0` to remove key counting limitations. **Default:** `1000`.
```

Comment:
I personally would not list any of these arguments here due to the the overhead of keeping them aligned with `querystring.parse()`. If missed while changing or adding something the documentation would be inconsistent.

I do not know how others feel about that though.